css/deck.css

```diff
@@ -7,3 +7,9 @@
 	background-image: url(../img/deck.svg);
 	filter: var(--background-invert-if-dark);
 }
+
+/* Add horizontal scrollbar for tables in card descriptions */
+.card-description table {
+	width: 100%; /* Ensure the table takes 100% width of the container */
+	overflow-x: auto; /* Enable horizontal scrolling */
+}
```

Comment:
I don't think we need this duplicate rule over here, best would be to only adjust this in the component that renders the table which would be here: https://github.com/nextcloud/deck/blob/9e41cb162de805677f50dff34874ef0430db47de/src/components/card/Description.vue#L52

Reply:
Note the scoping might limit the css rule so you probably need something like
```css
.description__text:deep(table) {
```